client/post-editor/media-modal/edit/edit-canvas.jsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' ),
+	ReactDOM = require( 'react-dom' );
```

Comment:
Camel-case variable naming applies to abbreviations ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions))